data/common.yaml

```diff
@@ -26,6 +26,7 @@ rsyslog::parser_priority: 45
 rsyslog::template_priority: 50
 rsyslog::filter_priority: 55
 rsyslog::action_priority: 60
+rsyslog::dont_linebreak_actions: false
```

Comment:
Don't add default value to Hiera if they are not changed from one OS to another.  This prevent puppet-string from displaying default-values when generating the module REFERENCE file.

Reply:
I did not know this.  Thanks for the explanation.  Wouldn't it be prudent, then to apply this policy to all of the pre-existing variables in common.yaml?

Reply:
IMHO it is a good idea, submitting this is a follow-up PR is fine.